### PR TITLE
fix: clients view search field

### PR DESCRIPTION
**Pull request by BLasan**
## Description
Client search using client name and filter the active and inactive accounts from the check bob completed.

## Related issues and discussion
#445 

## Screenshots, if any
Screenshot from 2020-03-25 14-55-56
Screenshot from 2020-03-25 14-56-03
Screenshot from 2020-03-25 14-56-07

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
@abhaychawla  Please review the PR

**Comment by OussEmaDevCode** [comment]
@BLasan Hey, make sure you change your commit message so that it applies to the [contribution guideline](https://github.com/openMF/web-app/blob/master/.github/COMMIT_MESSAGE.md) so that it looks something like this `feat: search clients feature`  and mention the issue as such: `Fixes: #683` in the description below instead of what you already put.

File: src/app/clients/clients.component.scss [hunk 1]
```diff
@@ -8,4 +8,4 @@
     margin: 4px;
     
   }
-}
+}
```
Comment (OussEmaDevCode) [inline src/app/clients/clients.component.scss]: remove all unnecessary changes.

**Comment by OussEmaDevCode** [comment]
Good job ! one more thing, make sure you mention the issue you're trying to fix in the commit description. (here's the [last commit](https://github.com/openMF/web-app/commit/793cc59899c56dc3713d62ac2a8a3b411c2340e9) as an example )

**Comment by BLasan** [comment]
Travis build error due to timeout

File: src/app/clients/clients.component.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div fxLayout="row" fxLayoutAlign="start center">
```
Comment (karantakalkar) [inline src/app/clients/clients.component.html:2]: @BLasan check the latest code for centers view https://github.com/openMF/web-app/pull/506/files, implement search feature in a similar manner, also can you rebase this pull request on latest code, it will be easy for me and you to compare.
